src/OpenColorIO/Op.cpp

```diff
@@ -149,10 +149,10 @@ OCIO_NAMESPACE_EXIT
 #ifdef OCIO_UNIT_TEST
 
 namespace OCIO = OCIO_NAMESPACE;
-#include "UnitTest.h"
-#include "MatrixOps.h"
-#include "LogOps.h"
-#include "NoOps.h"
+#include "unittest.h"
+#include "ops/Matrix/MatrixOps.h"
+#include "ops/Log/LogOps.h"
+#include "ops/NoOp/NoOps.h"
```

Comment:
**[Minor]** NoOp does not deserve to have its own directory. The original location was good at my point of view.

Reply:
I debated that too, but I think it might as well live with the other ops :P